### ISSUE TITLE
add more potential exitnode ips to home nodes

### ISSUE THREAD

**Issue by jhpoelen** [sudomesh/bugs#17]
Currently, home nodes are configured with a single exit node ip / port.

We'd like to include a list of ip addresses, so that we can run (or spin up) exitnodes behind IPs that we control.

**Comment by jhpoelen** [comment]
room.sudoroom.org: 142.254.26.9
sudoroom.org: 104.236.181.202
omnicommons.org: 104.131.158.147
peoplesopen.net: 107.170.219.5
exit1.sudomesh.org: 104.236.181.226

**Comment by jhpoelen** [comment]
Here's an example config from /etc/config/tunneldigger 

```
config broker 'main'
       list address '104.236.181.226:8942'
       list address '104.236.181.226:8943'
       list address '45.34.140.42:8942'
       list address '45.34.140.42:8943'
       list address '142.254.26.9:8942'
       list address '104.236.181.202:8942'
       list address '104.131.158.147:8942'
       list address '107.170.219.5:8942'
       option uuid 'XXXXXX'
       option interface 'l2tp0'
       option bind_to_interface 'eth1'
       option limit_bw_down '2048kbit'
       option limit_bw_up '2048kbit'
       option hook_script '/opt/mesh/tunnel_hook'
```

**Comment by paidforby** [comment]
Can be addressed in [makenode](https://github.com/sudomesh/makenode). The exit node IP is currently buried in [configs/config.js](https://github.com/sudomesh/makenode/blob/master/configs/config.js). I'd rather hardcode these new exit nodes into [/etc/config/tunneldigger](https://github.com/sudomesh/makenode/blob/master/configs/templates/files/etc/config/tunneldigger) because it seems more obvious to me and it will help in preparing for the deprecation of makenode, https://github.com/sudomesh/makenode/issues/19. Additionally, I'd like see options of exit nodes chosen through [peoplesopen-dash](https://github.com/sudomesh/peoplesopen-dash), not obscured inside of a js file.

**Comment by jhpoelen** [comment]
Sounds good to me. Ideally, we'd have a way to mesh the registry itself, allowing for exit nodes to join and leave a list of trusted exit nodes for a particular project like peoplesopen.net or somethingotheropen.net .

**Comment by paidforby** [comment]
remote patch for this bug is now available for TESTING, this should NOT be executed on the "real"  exitnode until further testing has be completed. Find the patch and instructions for execution in the makenode repo here, https://github.com/sudomesh/makenode/tree/master/patches

**Comment by gobengo** [comment]
If we need more exit nodes, here is a script to make more exitnodes: https://gitlab.ops.bengo.is/ben/mesh-network#digitalocean-exitnode

This week I'll test my exitnodes I made, and if they work I'll post an IP that we can add to the firmware if needed.

**Comment by jhpoelen** [comment]
@gobengo just noticed your exit node creation scripts - where you still thinking of merging that into the other exitnode creation script at https://github.com/sudomesh/exitnode ? I think it would be nice to have a single one to make it easier to test / improve. Curious to hear your thoughts on this.

**Comment by gobengo** [comment]
I cannot merge anything into that repo, but there has been a PR for 18
days. https://github.com/sudomesh/exitnode/pull/8

Traveling recently and until April. Have my nodes with me. Still need to
make time to test them against my exit node (and remember how).

Taking redeye tonight LA -> NYC. May be able to test this week but also
have some work to do out there and friends to see.

Stand by, help test, or merge what I got.

**Comment by jhpoelen** [comment]
Ok, I remember that pull request now.  I was hoping to get a more incremental set of improvements, but if you are confident that things'll work, then I'd be happy to pull the changes in.

**Comment by gobengo** [comment]
I cannot be confident without testing :)

Sounds best to wait a bit until I test. I will also make the PR smaller as
you wisely suggest.

**Comment by jhpoelen** [comment]
@gobengo ok, wishing you safe travels, and please let me know if you are all tested and confident. . .

**Comment by gobengo** [comment]
@jhpoelen Just looking at this now. I was wrong about PR 8 being my latest.
I forgot I got fed up (at the time) with the notion of building consensus between what works for me and what works for however this repo is governed.

I took what I had in my fork of this repo and made it better at https://github.com/gobengo/mesh-network ([github mirror](https://github.com/gobengo/mesh-network))

For now I closed #8 and made #11, which will at least add the `./build/remote <user@host>` command, as I think that's the smallest possible improvement here.

Once that's in, we can make moves toward pulling the entirety of [digitalocean-exitnode](https://gitlab.ops.bengo.is/ben/mesh-network/blob/master/bin/digitalocean-exitnode) et al (or someone can go port that over sooner. Gotta board plane now)
